.github/workflows/test.yml

```diff
@@ -16,32 +16,27 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
+      - uses: extractions/setup-just@v2
```

Comment:
I have been using [this](https://github.com/marketplace/actions/install-development-tools) instead, as it supports many more tools and is regularly updated, but totally up to you.:


```
- uses: taiki-e/install-action@v2
  with: { tool: just }
```

Reply:
Oh! Nice. I don't use GitHub actions much, so I'll switch.

Reply:
Just FYI if you use gitlab - they recently added github action support too, much nicer than the crappy single yaml system they have